### ISSUE TITLE
AttributeError: 'FrameShiftTruncation' object has no attribute 'aa_alt'

### ISSUE THREAD

**Issue by tavinathanson** [hammerlab/varcode#70]
Issue with multiple inheritance of `FrameShiftTruncation`, where `mutant_protein_sequence` uses the definition in `PrematureStop`?

```
/Users/tavi/drive/work/repos/cancer/topiary/topiary/mutant_epitope_predictor.pyc in predict(self, variant_collection, mhc_alleles, gene_expression_dict, min_gene_expresion_value, transcript_expression_dict, min_transcript_expresion_value, raise_on_variant_effect_error)
    145         fasta_dict = create_fasta_dict(
    146             effects=variant_effects.values(),
--> 147             padding_around_mutation=self.padding_around_mutation)
    148         return mhc_model.predict(fasta_dict)

/Users/tavi/drive/work/repos/cancer/topiary/topiary/mutant_epitope_predictor.pyc in create_fasta_dict(effects, padding_around_mutation)
     20         # TODO: will mhctools take an object key?
     21         key = effect.variant
---> 22         seq = effect.mutant_protein_sequence
     23         # some effects will lack a mutant protein sequence since
     24         # they are either silent or unpredictable

/Users/tavi/.virtualenvs/nejm/lib/python2.7/site-packages/memoized_property.pyc in fget_memoized(self)
     38     def fget_memoized(self):
     39         if not hasattr(self, attr_name):
---> 40             setattr(self, attr_name, fget(self))
     41         return getattr(self, attr_name)
     42 

/Users/tavi/drive/work/repos/cancer/varcode/varcode/effects.py in mutant_protein_sequence(self)
    546     def mutant_protein_sequence(self):
    547         prefix = self.original_protein_sequence[:self.aa_mutation_start_offset]
--> 548         return prefix + self.aa_alt
    549 
    550 

AttributeError: 'FrameShiftTruncation' object has no attribute 'aa_alt'
```

**Comment by iskandr** [comment]
Can you paste the variant this error came from? I'll add it to my
"problematic variants" unit test.

**Comment by iskandr** [comment]
Should be fixed in version `0.3`, but if you give me the variant I'll still add it to the unit tests.
